### ISSUE TITLE
Incorrectly formatted JSON data causes crash

### ISSUE THREAD

**Issue by AmpTheDev** [MediaModMC/MediaMod#8]
**Describe the bug**
When the browser handler receives a status update without any images, it'll still accept it and set the currently playing object. This causes an ArrayIndexOutOfBoundsException when rendering the player. If album art rendering is turned off, this doesn't happen, but the same issue occurs when no artists are found.
This is an issue because people could write malicious code that crashes your client as soon as it launches without you knowing.

**To Reproduce**
1. Launch the client and make sure the browser handler is enabled and ready.
2. Join a world or server to make sure that the player will render.
3. Send track JSON data without any image or artist data, for example:
```json
{
    "progress_ms":0,
    "item":{
        "duration_ms":0,
        "name":"Client Crasher",
        "album":{
            "artists":[

            ],
            "images":[

            ]
        }
    }
}
```
4. The client crashes while rendering

**Expected behavior**
The client refuses to render the malformed data or doesn't render any album art or artist text.

**Screenshots**
N/A

**Operating System**
Windows

**Service causing this bug, if applicable**
N/A

**Additional context**
None

**Comment by ConorTheDev** [comment]
This won't be an issue as the official services that MediaMod supports won't be returning any invalid json, closing.

**Comment by ConorTheDev** [comment]
Reopening as I only read the title, fixing issue now.

**Comment by AmpTheDev** [comment]
If you need the crash report, here it is: ```
---- Minecraft Crash Report ----

WARNING: coremods are present:
Contact their authors BEFORE contacting forge

// My bad.

Time: 9/21/19 7:49 PM
Description: Unexpected error

java.lang.ArrayIndexOutOfBoundsException: 0
	at me.conorthedev.mediamod.gui.PlayerOverlay.drawPlayer(PlayerOverlay.java:162)
	at me.conorthedev.mediamod.gui.PlayerOverlay.onRender(PlayerOverlay.java:122)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_7_PlayerOverlay_onRender_RenderGameOverlayEvent.invoke(.dynamic)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:49)
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:140)
	at net.minecraftforge.client.GuiIngameForge.pre(GuiIngameForge.java:847)
	at net.minecraftforge.client.GuiIngameForge.func_180479_a(GuiIngameForge.java:289)
	at net.minecraftforge.client.GuiIngameForge.func_175180_a(GuiIngameForge.java:120)
	at net.minecraft.client.renderer.EntityRenderer.func_181560_a(EntityRenderer.java:1396)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1051)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:349)
	at net.minecraft.client.main.Main.main(SourceFile:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at me.conorthedev.mediamod.gui.PlayerOverlay.drawPlayer(PlayerOverlay.java:162)
	at me.conorthedev.mediamod.gui.PlayerOverlay.onRender(PlayerOverlay.java:122)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_7_PlayerOverlay_onRender_RenderGameOverlayEvent.invoke(.dynamic)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:49)
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:140)
	at net.minecraftforge.client.GuiIngameForge.pre(GuiIngameForge.java:847)
	at net.minecraftforge.client.GuiIngameForge.func_180479_a(GuiIngameForge.java:289)
	at net.minecraftforge.client.GuiIngameForge.func_175180_a(GuiIngameForge.java:120)

-- Affected level --
Details:
	Level name: MpServer
	All players: 1 total; [EntityPlayerSP['AmpTheDev'/320, l='MpServer', x=199.98, y=70.00, z=261.20]]
	Chunk stats: MultiplayerChunkCache: 289, 289
	Level seed: 0
	Level generator: ID 00 - default, ver 1. Features enabled: false
	Level generator options: 
	Level spawn location: 202.00,64.00,257.00 - World: (202,64,257), Chunk: (at 10,4,1 in 12,16; contains blocks 192,0,256 to 207,255,271), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Level time: 31028 game time, 31028 day time
	Level dimension: 0
	Level storage version: 0x00000 - Unknown?
	Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
	Level game mode: Game mode: creative (ID 1). Hardcore: false. Cheats: false
	Forced entities: 96 total; [EntityCreeper['Creeper'/1546, l='MpServer', x=271.50, y=36.00, z=265.50], EntityZombie['Zombie'/524, l='MpServer', x=153.50, y=31.00, z=194.50], EntityZombie['Zombie'/1036, l='MpServer', x=139.50, y=28.00, z=187.50], EntityCreeper['Creeper'/1551, l='MpServer', x=230.50, y=36.00, z=209.50], EntityBat['Bat'/1042, l='MpServer', x=125.66, y=13.03, z=242.75], EntitySkeleton['Skeleton'/803, l='MpServer', x=174.50, y=24.00, z=250.50], EntitySkeleton['Skeleton'/548, l='MpServer', x=199.50, y=20.00, z=232.50], EntityCreeper['Creeper'/808, l='MpServer', x=273.50, y=57.00, z=259.50], EntityBat['Bat'/1067, l='MpServer', x=212.54, y=40.80, z=225.58], EntityBat['Bat'/1329, l='MpServer', x=128.10, y=42.56, z=325.70], EntityBat['Bat'/1330, l='MpServer', x=241.44, y=16.27, z=212.93], EntityZombie['Zombie'/1079, l='MpServer', x=131.50, y=58.00, z=336.50], EntityCreeper['Creeper'/569, l='MpServer', x=216.50, y=38.00, z=255.50], EntityPlayerSP['AmpTheDev'/320, l='MpServer', x=199.98, y=70.00, z=261.20], EntitySkeleton['Skeleton'/1599, l='MpServer', x=139.50, y=27.00, z=194.50], EntityZombie['Zombie'/68, l='MpServer', x=125.50, y=33.00, z=195.50], EntityRabbit['Rabbit'/70, l='MpServer', x=122.56, y=72.00, z=199.31], EntitySpider['Spider'/71, l='MpServer', x=121.69, y=20.00, z=251.31], EntityChicken['Chicken'/72, l='MpServer', x=120.50, y=89.00, z=278.50], EntityChicken['Chicken'/73, l='MpServer', x=120.50, y=90.00, z=277.50], EntityChicken['Chicken'/74, l='MpServer', x=123.50, y=88.00, z=277.50], EntityChicken['Chicken'/75, l='MpServer', x=124.50, y=88.00, z=276.50], EntitySpider['Spider'/333, l='MpServer', x=178.50, y=20.00, z=262.50], EntityZombie['Zombie'/89, l='MpServer', x=130.34, y=23.00, z=187.34], EntitySkeleton['Skeleton'/1625, l='MpServer', x=155.50, y=22.00, z=216.50], EntityCreeper['Creeper'/90, l='MpServer', x=140.03, y=19.00, z=216.41], EntityZombie['Zombie'/93, l='MpServer', x=131.50, y=13.00, z=226.50], EntityCreeper['Creeper'/95, l='MpServer', x=143.00, y=29.00, z=219.66], EntityChicken['Chicken'/96, l='MpServer', x=141.69, y=86.00, z=274.50], EntityBat['Bat'/97, l='MpServer', x=135.19, y=16.03, z=304.66], EntityCow['Cow'/106, l='MpServer', x=148.50, y=66.00, z=189.69], EntityCow['Cow'/107, l='MpServer', x=148.44, y=65.00, z=188.31], EntityCow['Cow'/108, l='MpServer', x=147.50, y=66.00, z=186.50], EntityCreeper['Creeper'/109, l='MpServer', x=152.50, y=22.00, z=202.50], EntityCow['Cow'/110, l='MpServer', x=150.72, y=67.00, z=205.53], EntitySkeleton['Skeleton'/111, l='MpServer', x=148.88, y=18.00, z=241.38], EntityChicken['Chicken'/112, l='MpServer', x=145.50, y=89.00, z=268.50], EntityChicken['Chicken'/113, l='MpServer', x=154.38, y=86.00, z=274.44], EntityChicken['Chicken'/114, l='MpServer', x=147.03, y=88.00, z=270.22], EntityRabbit['Rabbit'/115, l='MpServer', x=143.28, y=84.00, z=258.50], EntityRabbit['Rabbit'/116, l='MpServer', x=149.09, y=90.00, z=264.25], EntityRabbit['Rabbit'/117, l='MpServer', x=148.19, y=90.00, z=264.19], EntityItem['item.item.egg'/118, l='MpServer', x=147.13, y=89.00, z=268.53], EntityZombie['Zombie'/119, l='MpServer', x=157.16, y=63.00, z=326.56], EntitySkeleton['Skeleton'/377, l='MpServer', x=126.19, y=12.22, z=228.69], EntitySkeleton['Skeleton'/121, l='MpServer', x=158.88, y=60.00, z=328.47], EntitySkeleton['Skeleton'/126, l='MpServer', x=174.06, y=21.00, z=236.47], EntityCow['Cow'/131, l='MpServer', x=182.19, y=66.00, z=188.72], EntityCow['Cow'/132, l='MpServer', x=181.34, y=67.00, z=204.47], EntityCow['Cow'/133, l='MpServer', x=179.25, y=66.00, z=193.25], EntityBat['Bat'/134, l='MpServer', x=191.34, y=29.10, z=212.63], EntitySquid['Squid'/136, l='MpServer', x=186.91, y=59.69, z=219.66], EntityCow['Cow'/145, l='MpServer', x=200.72, y=67.00, z=183.53], EntityCow['Cow'/146, l='MpServer', x=203.81, y=70.00, z=184.97], EntityCow['Cow'/147, l='MpServer', x=202.50, y=67.00, z=184.50], EntityCow['Cow'/148, l='MpServer', x=193.91, y=65.00, z=198.97], EntitySquid['Squid'/151, l='MpServer', x=200.00, y=61.78, z=223.72], EntitySquid['Squid'/152, l='MpServer', x=190.43, y=59.32, z=235.46], EntityChicken['Chicken'/154, l='MpServer', x=187.47, y=70.00, z=270.91], EntityChicken['Chicken'/155, l='MpServer', x=195.13, y=69.00, z=279.63], EntityChicken['Chicken'/156, l='MpServer', x=194.16, y=69.00, z=283.91], EntityItem['item.item.egg'/157, l='MpServer', x=196.91, y=71.00, z=283.00], EntityItem['item.item.egg'/158, l='MpServer', x=193.91, y=69.00, z=284.31], EntityBat['Bat'/1438, l='MpServer', x=163.28, y=26.31, z=218.69], EntityChicken['Chicken'/159, l='MpServer', x=187.44, y=70.00, z=277.13], EntityItem['item.item.egg'/160, l='MpServer', x=193.13, y=73.00, z=291.28], EntityChicken['Chicken'/161, l='MpServer', x=195.50, y=64.00, z=308.50], EntityChicken['Chicken'/162, l='MpServer', x=202.41, y=64.00, z=301.59], EntityChicken['Chicken'/163, l='MpServer', x=196.28, y=64.00, z=309.31], EntityChicken['Chicken'/164, l='MpServer', x=194.50, y=64.00, z=308.50], EntityRabbit['Rabbit'/165, l='MpServer', x=201.50, y=65.19, z=317.78], EntityRabbit['Rabbit'/166, l='MpServer', x=192.84, y=65.00, z=310.13], EntityRabbit['Rabbit'/167, l='MpServer', x=195.28, y=65.00, z=313.22], EntityCow['Cow'/178, l='MpServer', x=209.75, y=66.00, z=186.31], EntitySquid['Squid'/179, l='MpServer', x=197.81, y=56.00, z=225.66], EntitySpider['Spider'/435, l='MpServer', x=242.50, y=35.00, z=277.50], EntityBat['Bat'/181, l='MpServer', x=208.25, y=21.10, z=204.25], EntityZombie['Zombie'/183, l='MpServer', x=218.50, y=35.00, z=275.50], EntityChicken['Chicken'/187, l='MpServer', x=235.44, y=79.00, z=283.34], EntitySquid['Squid'/1213, l='MpServer', x=180.34, y=62.00, z=218.69], EntityCreeper['Creeper'/1470, l='MpServer', x=151.50, y=26.00, z=219.50], EntityChicken['Chicken'/191, l='MpServer', x=241.50, y=77.00, z=281.50], EntityChicken['Chicken'/192, l='MpServer', x=234.50, y=78.00, z=277.00], EntityChicken['Chicken'/193, l='MpServer', x=244.47, y=77.00, z=291.75], EntitySkeleton['Skeleton'/455, l='MpServer', x=221.50, y=44.00, z=319.50], EntityBat['Bat'/202, l='MpServer', x=270.63, y=21.13, z=216.31], EntityZombie['Zombie'/204, l='MpServer', x=269.50, y=60.00, z=266.06], EntitySkeleton['Skeleton'/460, l='MpServer', x=266.50, y=30.00, z=294.50], EntitySpider['Spider'/205, l='MpServer', x=271.50, y=38.00, z=286.50], EntityCreeper['Creeper'/206, l='MpServer', x=259.28, y=64.00, z=283.53], EntitySpider['Spider'/1233, l='MpServer', x=139.50, y=20.00, z=190.50], EntitySkeleton['Skeleton'/1494, l='MpServer', x=259.50, y=28.00, z=184.50], EntityZombie['Zombie'/223, l='MpServer', x=275.50, y=13.00, z=218.50], EntityCreeper['Creeper'/993, l='MpServer', x=256.50, y=53.00, z=190.50], EntitySkeleton['Skeleton'/1256, l='MpServer', x=133.50, y=30.00, z=189.50], EntitySkeleton['Skeleton'/1257, l='MpServer', x=194.50, y=26.00, z=241.06]]
	Retry entities: 0 total; []
	Server brand: fml,forge
	Server type: Integrated singleplayer server
Stacktrace:
	at net.minecraft.client.multiplayer.WorldClient.func_72914_a(WorldClient.java:417)
	at net.minecraft.client.Minecraft.func_71396_d(Minecraft.java:2536)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:378)
	at net.minecraft.client.main.Main.main(SourceFile:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

-- System Details --
Details:
	Minecraft Version: 1.8.9
	Operating System: Windows 10 (amd64) version 10.0
	CPU: 4x Intel(R) Core(TM) i7-7500U CPU @ 2.70GHz
	Java Version: 1.8.0_51, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 545554032 bytes (520 MB) / 1040187392 bytes (992 MB) up to 2147483648 bytes (2048 MB)
	JVM Flags: 8 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx2G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	IntCache: cache: 0, tcache: 0, allocated: 12, tallocated: 94
	FML: MCP 9.19 Powered by Forge 11.15.1.2318 Optifine OptiFine_1.8.9_HD_U_I7 4 mods loaded, 4 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHIJAAAA	mcp{9.19} [Minecraft Coder Pack] (minecraft.jar) 
	UCHIJAAAA	FML{8.0.99.99} [Forge Mod Loader] (forge-1.8.9-11.15.1.2318-1.8.9.jar) 
	UCHIJAAAA	Forge{11.15.1.2318} [Minecraft Forge] (forge-1.8.9-11.15.1.2318-1.8.9.jar) 
	UCHIJAAAA	mediamod{0.0.1} [MediaMod] (MediaMod-0.0.1.jar) 
	Loaded coremods (and transformers): 
	GL info: ' Vendor: 'NVIDIA Corporation' Version: '4.6.0 NVIDIA 436.30' Renderer: 'GeForce 940MX/PCIe/SSE2'
	Launched Version: 1.8.9-forge1.8.9-11.15.1.2318-1.8.9
	LWJGL: 2.9.4
	OpenGL: GeForce 940MX/PCIe/SSE2 GL version 4.6.0 NVIDIA 436.30, NVIDIA Corporation
	GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fml,forge'
	Type: Client (map_client.txt)
	Resource Packs: §b§l1.14 §r§bdefault textures§r.zip
	Current Language: English (US)
	Profiler Position: N/A (disabled)
	CPU: 4x Intel(R) Core(TM) i7-7500U CPU @ 2.70GHz
```
When album art rendering is enabled and there is artist information, the line number changes from 162 to 164.

**Comment by AmpTheDev** [comment]
wow scam it didn't format correctly
